src/cio.ts

```diff
@@ -180,6 +277,33 @@ export async function identifyUser({
   }
 }
 
+export async function syncNotificationFlagsToCio({
+  userId,
+  notificationFlags,
+}: {
+  userId: string;
+  notificationFlags: User['notificationFlags'];
+}): Promise<void> {
+  if (process.env.NODE_ENV === 'development') {
+    return;
+  }
+
+  const cioTopics = getNotificationFlagsCioTopics(notificationFlags);
+
+  try {
+    await cio.identify(userId, cioTopics);
+  } catch (err) {
+    if (err instanceof CustomerIORequestError && err.statusCode === 400) {
+      logger.warn(
```

Comment:
```suggestion
      logger.error(
```

and let's monitor for this please